spec/workers/detect_invariants_spec.rb

```diff
@@ -107,5 +107,32 @@
         ),
       )
     end
+
+    it 'detects submitted applications with more than three course choices' do
+      good_application_form = create(:completed_application_form, submitted_application_choices_count: 3)
+      bad_application_form = create(:completed_application_form, submitted_application_choices_count: 4)
+
+      DetectInvariants.new.perform
+
+      expect(Raven).to have_received(:capture_exception).with(
+        DetectInvariants::SubmittedApplicationHasMoreThanThreeChoices.new(
+          <<~MSG,
+            The following application forms have been submitted with more than three course choices
+
+            http://localhost:3000/support/applications/#{bad_application_form.id}
+          MSG
+        ),
+      )
+
+      expect(Raven).not_to have_received(:capture_exception).with(
```

Comment:
Nit: could express more or less the same idea (ie—"Raven doesn't receive anything related to the good_application") by replacing this second expectation with:

`expect(Raven).to have_received(:capture_exception).once`